src/cpx_vars.jl

```diff
@@ -78,6 +78,22 @@ function add_var!(model::Model, constridx::Vector, constrcoef::Vector, l::Vector
     return add_var!(model, ivec(constridx), fvec(constrcoef), fvec(l), fvec(u), fvec(objcoef))
 end
 
+function get_varLB(model::Model, col::Cint)
```

Comment:
Style: `getlb`

Reply:
I think we should keep the name `get_lb` for the function that does exactly the same as `cpxgetlb`, namely getting lowerbounds for a range of columns and not for just one column.